src/App.js

```diff
@@ -1,23 +1,12 @@
-import logo from './logo.svg';
-import './App.css';
-import React from 'react'
+import "./App.css";
+import React from "react";
+import AstronautsTable from "./AstronautsTable";
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React on Grace's app
-        </a>
+        <AstronautsTable />
       </header>
     </div>
   );
```

Comment:
Let's take a look at creating 2 folders in `src`. `components` & `tests` and then move files where needed. The only component at your `src` level should be `App`. This will help things stay organized as you grow. You wouldn't need it specifically for this project, but its good practice. Each project will be organized a little differently depending on who originally set it up, but you should never see a project all dumped into the same folder. If you do you have an easy starting point to make an impact :-D.